generators/kubernetes/templates/kubectl-apply.sh.ejs

```diff
@@ -47,31 +63,55 @@ logSummary() {
     echo "#####################################################"
 }
 
-<%_ if (kubernetesNamespace !== 'default') { _%>
-kubectl apply -f namespace.yml
-<%_ } _%> <%_ if (serviceDiscoveryType === 'eureka' || serviceDiscoveryType === 'consul') { _%>
-kubectl apply -f registry/
-<%_ } _%> <%_ if (istio) { _%>
-kubectl label namespace <%- kubernetesNamespace %> istio-injection=enabled --overwrite=true
-<%_ } _%> <%_ appConfigs.forEach((appConfig, index) =>  { _%>
-kubectl apply -f <%- appConfig.baseName.toLowerCase() %>/
-<%_ }) _%>
-<%_ if (useKafka === true) { _%>
-kubectl apply -f messagebroker/
-<%_ } _%> <%_ if (monitoring === 'elk') { _%>
-kubectl apply -f console/
-<%_ } _%> <%_ if (monitoring === 'prometheus') { _%>
-kubectl apply -f monitoring/jhipster-prometheus-crd.yml
-until [ $(kubectl get crd prometheuses.monitoring.coreos.com 2>>/dev/null | wc -l) -ge 2 ]; do
-    echo "Waiting for the custom resource prometheus operator to get initialised";
-    sleep 5;
+default() {
+    <%_ if (kubernetesNamespace !== 'default') { _%>
+    kubectl apply -f namespace.yml
+    <%_ } _%> <%_ if (serviceDiscoveryType === 'eureka' || serviceDiscoveryType === 'consul') { _%>
+    kubectl apply -f registry/
+    <%_ } _%> <%_ if (istio) { _%>
+    kubectl label namespace <%-kubernetesNamespace%> istio-injection=enabled --overwrite=true
+    <%_ } _%> <%_ appConfigs.forEach((appConfig, index) =>  { _%>
+    kubectl apply -f <%- appConfig.baseName.toLowerCase() %>/
+    <%_ }) _%>
+    <%_ if (useKafka === true) { _%>
+    kubectl apply -f messagebroker/
+    <%_ } _%> <%_ if (monitoring === 'elk') { _%>
+    kubectl apply -f console/
+    <%_ } _%> <%_ if (monitoring === 'prometheus') { _%>
+    kubectl apply -f monitoring/jhipster-prometheus-crd.yml
+    until [ $(kubectl get crd prometheuses.monitoring.coreos.com 2>>/dev/null | wc -l) -ge 2 ]; do
+        echo "Waiting for the custom resource prometheus operator to get initialised";
+        sleep 5;
+    done
+    kubectl apply -f monitoring/jhipster-prometheus-cr.yml
+    kubectl apply -f monitoring/jhipster-grafana.yml
+    kubectl apply -f monitoring/jhipster-grafana-dashboard.yml
+    <%_ } _%>
+
+    <%_ if (istio && kubernetesServiceType === 'Ingress') { _%>
+    kubectl apply -f istio/
+    <%_ } _%>
+}
+
+kustomize() {
+    kubectl apply -k ./
+}
+
+scaffold() {
```

Comment:
skaffold

Reply:
No, that's the shell script function name. I'm calling skaffold inside that function (to avoid name conflict).